services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
```

Comment:
It's a good practice to make functions such as this one return something.

Reply:
So, for example, we can return an Order, but the task does not say anything about the return. I think the return is unnecessary here cause we can use `get_orders` to take information about order.

Reply:
I agree that instructions don't say anything about it, but to use `get_orders()` means to make an additional query to the database. And it's best to avoid unnecessary database interactions when possible. :)